tests/codeception/functional/Paragraphs/SpacerCest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * Class SpacerCest.
+ *
+ * @group paragraphs
+ * @group spacer
+ */
+class SpacerCest {
+
+    /**
+     * The spacer paragraph has one custom field, to set the size of the bottom margin.
+     *
+     */
+
+    public function testSpacerParagraph(FunctionalTester $I){
```

Comment:
Do you have your code standards incorrectly set? this should be 2 space tabs as well.

Reply:
I did a vscode update that ended up changing some preferences.  Working on getting it set correctly again.